sdk/identity/identity/test/integration/azureFunctionsTest.spec.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ServiceClient } from "@azure/core-client";
+import { createPipelineRequest } from "@azure/core-rest-pipeline";
+import { assert } from "chai";
+import { Context } from "mocha";
+import { isLiveMode } from "@azure-tools/test-recorder";
+
+
+describe.only("AzureFunctions Integration test", function() {
+    it("test the Azure Functions endpoint where the sync MI credential is used.", async function(this: Context) {
+        if(!isLiveMode){
+            this.skip();
+        }
+        const baseUri = baseUrl()
+        const client = new ServiceClient({baseUri: baseUri})
+        const pipelineRequest = createPipelineRequest({
+            url: baseUri,
+            method: "GET"
+        })
+        const response = await client.sendRequest(pipelineRequest);
+        assert.equal(response.status,200,`Expected status 200. Received ${response.status}`);
```

Comment:
I think there's some prettier formatting that needs to happen here - do you have that configured in vscode? If not, we can set that up tomorrow if you'd like?